controllers/storagecluster/cephcluster.go

```diff
@@ -306,6 +312,13 @@ func newCephCluster(sc *ocsv1.StorageCluster, cephImage string, nodeCount int, s
 				"mon":     getPlacement(sc, "mon"),
 				"arbiter": getPlacement(sc, "arbiter"),
 			},
+			PriorityClassNames: rook.PriorityClassNamesSpec{
+				"all":         openshiftUserCritical,
+				cephv1.KeyMds: openshiftUserCritical,
```

Comment:
Instead of specifying the MDS key, you will need to set this on the Filesystem CR under the metadataServer settings